### DISCUSSION TITLE
We can select and copy the version from About page

### DISCUSSION THREAD

**Discussion by cjmling** [spacedriveapp/spacedrive#1530] [Feature Request]
At present it is not selectable on Windows 11. It will be handy when reporting issue.

![RjoIuxSBSA](https://github.com/spacedriveapp/spacedrive/assets/1612187/6e05d88d-65cc-4d5a-b16a-cffe1b0323fd)

**Comment by utkubakir** [comment]
Well since we are releasing alpha builds now, you would only need to give us the "v0.1.0" part and that shouldn't be too hard to write :D
